src/SampleWebApp/SeedData/Models/AllNoxTypeModel.cs

```diff
@@ -10,5 +10,66 @@ public class AllNoxTypeModel
         public string TextField { get; set; } = string.Empty;
         public string VatNumber { get; set; } = string.Empty;
+        public short CountryNumber { get; set;} = short.MinValue;
+        public short CurrencyNumber { get; set; } = short.MinValue;
+        public string CultureCode { get; set; } = string.Empty;
+        public double AreaValue { get; set; } = double.MinValue;
+        public string AreaUnit { get; set; } = "SquareMeter";
+        public double DistanceValue { get; set; } = double.MinValue;
+        public string DistanceUnit { get; set; } = "Kilometer";
+        public string Email { get; set; } = string.Empty;
+        public EncryptedTextTypeOptions EncryptedTextTypeOptions { get; set; } = null!;
+        public string FileName { get; set; } = string.Empty;
+        public string FileUrl { get; set; } = string.Empty;
+        public ulong FileSize { get; set; } = ulong.MinValue;
+        public string HtmlText { get; set; } = string.Empty;
+        public string ImageUrl { get; set; } = string.Empty;
+        public string ImageName { get; set; } = string.Empty;
+        public ulong ImageSize { get; set; } = ulong.MinValue;
+        public string InternetDomain { get; set; } = string.Empty;
+        public string IpAddress { get; set; } = string.Empty;
+        public string Json { get; set; } = string.Empty;
+        public string JwtToken { get; set; } = string.Empty;
+        public string LanguageCode { get; set; } = string.Empty;
+        public double Latitude { get; set; } = 0;
+        public double Longitude { get; set; } = 0;
+        public string TranslatedText { get; set; } = string.Empty;
+        public string CurrencyCode { get; set; } = string.Empty;
+        public decimal MoneyAmount { get; set; } = decimal.MinValue;
+        public string MacAddress { get; set; } = string.Empty;
+        public string Markdown { get; set; } = string.Empty;
+        public byte Month { get; set; } = 1;
+        public float Percentage { get; set; } = float.MinValue;
+        public decimal Number { get; set; } = decimal.MinValue;
+        public NumberTypeOptions NumberTypeOptions { get; set; } = null!;
+        public string TimeZoneCode { get; set; } = string.Empty;
+        public decimal Temperature { get; set; } = decimal.MinValue;
+        public string TemperatureUnit { get; set; } = string.Empty;
+        public decimal Volume { get; set; } = decimal.MinValue;
+        public string VolumeUnit { get; set; } = string.Empty;
+        public decimal Weight { get; set; } = decimal.MinValue;
+        public string WeightUnit { get; set; } = string.Empty;
+        public string Uri { get; set; } = string.Empty;
+        public string Url { get; set; } = string.Empty;
+        public string Yaml { get; set; } = string.Empty;
+        public ushort Year { get; set; } = ushort.MinValue;
+
+        public StreetAddressItem StreetAddressItem { get; set; } = null!;
```

Comment:
Can we rename the property to StreetAddress?

Reply:
done